samples/matter/lock/src/chip_project_config.h

```diff
@@ -14,3 +14,14 @@
  */
 
 #pragma once
+
+#ifdef CONFIG_THREAD_WIFI_SWITCHING
+/*
+ * Reduce the code size by disabling ZCL progress level logs.
+ * Door Lock cluster logs are extremely verbose and significantly increase the code size.
+ */
+#define CHIP_CONFIG_LOG_MODULE_Zcl_PROGRESS 0
```

Comment:
```suggestion
#define CHIP_CONFIG_LOG_MODULE_ZCL_PROGRESS 0
```
???

Reply:
No, `Zcl` is the name of the logging module in Matter so this cannot be uppercase :)